src/test/kotlin/com/metalbear/mirrord/MirrordPluginTest.kt

```diff
@@ -168,8 +136,32 @@ internal class MirrordPluginTest {
                 }
             }
 
-            step("Start Debugging") {
+            step("Create config file") {
+                waitFor(ofSeconds(30)) {
+                    createMirrordConfig.isShowing
+                }
+                dumbAware {
+                    createMirrordConfig.click()
+                }
+                editorTabs {
+                    waitFor {
+                        isFileOpened("mirrord.json")
+                    }
+                    // switch back to app.py
+                    findText("app.py").click()
```

Comment:
This shouldn't be done here, but rather just when you need it - before setting the breakpoint